doc/releases/release-notes-1.14.rst

```diff
@@ -159,7 +163,41 @@ Build and Infrastructure
 Libraries / Subsystems
 ***********************
 
-* TBD
+* Logging:
+
+  - Removed sys_log which has been replaced by new logging subsystem introduced
+    in v1.13
+  - Refactored log modules registeration macros.
+  - Improved synchronous operation (see :option:`CONFIG_LOG_IMMEDIATE`).
+  - Added commands to control the logger using shell.
+  - Added :c:macro:`LOG_PANIC()` call to the fault handlers to ensure that logs
+    are outputed on fault.
+  - Added mechanism for handling logging of transient strings. See
+    :cpp:func:`log_strdup`.
+  - Added support for up to 15 arguments in the log message.
+  - Added optional function name prefix in the log message.
+  - Changed logging thread priority to the lowest application priority.
+  - Added notification about dropped log messages due to insufficent logger
+    buffer size.
+  - Added RTT log backend.
+  - Added native_posix log backend.
+  - Added net log backend.
+  - Added SWO log backend.
+  - Added Xtensa Sim log backend.
+  - Changed default timestamp source function to :cpp:func:`k_uptime_get_32`.
+
+* Shell:
+
+  - Added new implementation replacing existing one. See :ref:`shell_label`.
+  - Added UART backend.
+  - Added RTT backend.
+  - Added telnet backend.
+
```

Comment:
dummy?

Reply:
that's shell right? That's kind of a shell feature rather than real backend. I've added reference to shell documentation in the note about new shell implementation. Didn't want to repeat shell features here.